provider/bigquery/bigquery.go

```diff
@@ -10,15 +10,13 @@ import (
 	"google.golang.org/api/option"
 )
 
-// Client is wrapper for bigquery client
-type Client struct {
+type bigQueryClient struct {
```

Comment:
if possible, since the package name is already bigquery, we could probably just use `client` instead of `bigQueryClient`.

but since it is a private struct, it should be okay since only the package knows about it, you can decide on this.

ref: https://blog.golang.org/package-names#TOC_3.

Reply:
the reason I renaming it to `bigqueryClient` is that I created another client named `iamClient`, to make them look as they are in the same position as clients